src/utils/container-runtime/src/runtime.rs

```diff
@@ -508,8 +514,48 @@ impl ContainerRuntime {
     }
 }
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
 #[derive(Copy, Clone)]
 pub enum Signal {
     TERM,
     KILL,
 }
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[cfg(not(target_os = "linux"))]
+fn is_selinux_present() -> bool {
+    false
+}
+
+#[cfg(target_os = "linux")]
+fn is_selinux_present() -> bool {
```

Comment:
Should we add SELinux detection to `kamu system info`? Or is it already included there via docker/podman info?

Reply:
> re `z` & `Z` labels

Yeah, I haven't seen them since I started using Fedora either:

- https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
- https://github.com/containers/podman/blob/main/troubleshooting.md#2-cant-use-volume-mount-get-permission-denied

--

> Should we add SELinux detection to kamu system info? Or is it already included there via docker/podman info?

No, this information is already contained in output

> docker (since it's the default)

```
➜  ~ kamu system info | grep -i selinux
            ...
            "org.opencontainers.runtime-spec.features": ... \"selinux\":{\"enabled\":true}} ...
            ...
```

> podman
```
➜   kamu config get engine.runtime
podman

➜  ~ kamu system info | grep -i selinux
...
          "version": "crun version 1.15\ncommit: e6eacaf4034e84185fd8780ac9262bbf57082278\nrundir: /run/user/1000/crun\nspec: 1.0.0\n+SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +LIBKRUN +WASM:wasmedge +YAJL"
...
          "selinuxEnabled": true
...
```

Reply:
Thanks for checking!